#
# gcc-multilib64dir.diff is always applied by rules.patch
#
# "debian/patches/series" generated by "./debian/rules patch"
# in do_patch_srcpkg sometimes includes "gcc-multilib64dir",
# sometimes doesn't. (e.g. gcc-cross-in* and gcc-runtime).
# "gcc-multilib64dir" should be always applied because
# "64bithack.patch" override a file modified by it.
#
# NOTE: This patch must be applied BEFORE do_patch_srcpkg
#
# Skerlet <debian-squeeze@swc.toshiba.co.jp>
#
--- a/debian/rules.patch	2014-04-22 14:36:40.621072810 +0900
+++ b/debian/rules.patch	2014-04-22 14:37:34.965059927 +0900
@@ -308,12 +308,12 @@
   ifeq (,$(findstring libjava, $(biarch_multidir_names)))
     debian_patches += libjava-nobiarch-check
   endif
-  debian_patches += config-ml gcc-multilib64dir
+  debian_patches += config-ml
 endif
 
-ifneq (,$(findstring sparc64,$(DEB_TARGET_ARCH)))
-  debian_patches += gcc-multilib64dir
-endif
+# debian-squeeze: gcc-multilib64dir should be always applied because
+# an essential poky patch override a file modified by it
+debian_patches += gcc-multilib64dir
 
 ifeq ($(biarchn32)-$(biarch64),yes-yes)
   ifneq (,$(findstring $(DEB_TARGET_ARCH),mips mipsel))
